packages/@dcl/inspector/src/components/Tree/Tree.tsx

```diff
@@ -67,7 +67,7 @@ function Tree<T>(props: Props<T>) {
   const quitInsertMode = () => setInsertMode(false)
 
   const handleToggleExpand = (_: React.MouseEvent) => {
-    onToggle(value, !open)
+    onToggle(value, !selected || !open)
```

Comment:
This is not related to the issue that this PR fixes, but I took the chance to solve it because it was very annoying: every time you selected a different entity in the tree, it would close it (toggle it). Now it only toggles when you click the currently selected entity.